examples/binaries/proxy-reservation-with-gas/src/lib.rs

```diff
@@ -42,12 +43,13 @@ mod wasm {
 
     static mut DESTINATION: ActorId = ActorId::new([0u8; 32]);
     static mut DELAY: u32 = 0;
+    static mut RESERVATION_AMOUNT: u64 = 0;
 
     #[no_mangle]
     extern "C" fn handle() {
         let gas_limit: u64 = msg::load().expect("Failed to decode `gas_limit: u64'");
-        let reservation_id =
-            ReservationId::reserve(6_000_000_000, 80).expect("Failed to reserve gas");
+        let reservation_id = ReservationId::reserve(unsafe { RESERVATION_AMOUNT }, 80)
```

Comment:
don't use 0
go with something like 10k gas and huge delay
such less amount will be banned in other actual PR

Reply:
it's set with init msg. Discussed